paimon-common/src/test/java/org/apache/paimon/sort/zorder/TestZOrderByteUtil.java

```diff
@@ -108,7 +109,7 @@ public void testInterleaveRandomExamples() {
 
             String stringResult = interleaveStrings(testStrings);
 
-            Assert.assertEquals(
+            assertEquals(
```

Comment:
Can you modify these junit4 assertion to juni5 assertThat?

Reply:
Thanks for your review, Of course.

Reply:
@JingsongLi  done.